external/storm-kafka/src/jvm/storm/kafka/PartitionManager.java

```diff
@@ -126,7 +127,7 @@ public Map getMetricsDataMap() {
     }
 
     //returns false if it's reached the end of current batch
-    public EmitState next(SpoutOutputCollector collector) {
+    public EmitState next(SpoutOutputCollector collector, String emitStreamId) {
```

Comment:
what is the need of additional argument since spoutConfig is available to this class. stream name can be initialized once in the prepare phase itself.